### DISCUSSION TITLE
Some installation Errors

### DISCUSSION THREAD

**Discussion by Mechaboy95** [carson-katri/dream-textures#56] [Q&A] [answered]
After adding the addon to blender (through the main install button), I installed the dependencies like AskNK's video instructed, however after 
'Downloading torch-1.11.0-cp310-win_amd64.whl' I received 17 'INFO' 
with the last one reading 
'INFO: pip is looking at multiple versions of albumentations to determine which version is compatible with other requirements. This could take a while.'
After this I had my first error 
'ERROR: Could not find a version that satisfies the requirement basicsr>=1.4.2 (from gfpgan) (from versions: 1.3.1, 1.3.2 ... 1.4.1, 1.4.2)'
and then
'ERROR: No matching distribution found for basicsr>=1.4.2'
then a warning
'WARNING: You are using pip version 21.2.4; however, version 22.2.2 is available.
You should consider upgrading via the 'C:\Program Files\Blender Foundation\Blender 3.3\3.3\python\bin\python.EXE -m pip install --upgrade pip' command.'

Can someone please help me as I am unsure of how to fix this problem

My terminal:
![Error](https://user-images.githubusercontent.com/88431933/191625839-36aa8351-cefc-4fb7-abb9-a4192be04755.PNG)

**Comment by carson-katri** [comment, marked as answer]
Take a look at the Troubleshooting section of the addon about basicsr, or the note in the readme about windows installation:

* Note for Windows users - If you get an error when installing dependencies that looks similar to the one described https://github.com/carson-katri/dream-textures/issues/13, your dependency file paths might be too long (can't be longer than 256 characters). You can solve it by telling Windows to allow long file paths in the registry:
    * Open up the Window registry (Start > Run > regedit)
    * Navigate to `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem`
    * Set `LongPathsEnabled` to `1`

**Reply by Mechaboy95** [reply]
That solution is so simple
and it solves my 'error' 
but I now have ALLOT of warnings
![warning](https://user-images.githubusercontent.com/88431933/191637095-c287f76a-9dc6-4ad8-a6d6-da09b93a6212.PNG)
![warning2](https://user-images.githubusercontent.com/88431933/191637100-e51891fc-c68d-48b3-b6c7-53d904f27ffd.PNG)

**Reply by carson-katri** [reply]
As long as there are no errors, everything should install ok.

**Reply by Mechaboy95** [reply]
There is an OSError, but its not in red
![image](https://user-images.githubusercontent.com/88431933/191637508-3918f7a2-8286-45b0-99be-e2a0800be0d0.png)

**Reply by carson-katri** [reply]
Ok, you might want to try starting from a fresh Blender install. There is an issue open with a similar error you may want to look at as well.

**Reply by Mechaboy95** [reply]
Can I just repair my installation (I want to keep my preferences, startup interface and addons) or should I completely remove blender and re-install it

**Reply by carson-katri** [reply]
Perhaps try deleting the `pip-install-xxx` directory it mentions in the error and run it again.

**Reply by nikolatesla20** [reply]
Is it possible to add this registry key to the install script so it just works?
